### DISCUSSION TITLE
man: man -Tpdf fails

### DISCUSSION THREAD

**Discussion by vbauerster** [netblue30/firejail#6142] [Q&A] [answered]
I'm trying to convert a man page to a pdf, following command fails when firejail is enabled:

```
❯ man -Tpdf find > man-find.pdf
groff: error: couldn't exec gropdf: Permission denied
/usr/bin/man: command exited with status 4: (cd /usr/share/man && /usr/lib/man-db/zsoelim) | (cd /usr/share/man && /usr/lib/man-db/manconv -f UTF-8:ISO-8859-1 -t UTF-8//IGNORE) | (cd /usr/share/man && preconv -e UTF-8) | (cd /usr/share/man && tbl) | (cd /usr/share/man && groff -mandoc -Tpdf)
```

Any suggestion what needs to be done?

**Comment by ghost** [comment, marked as answer]
On my Arch Linux box `/usr/bin/gropdf` is a `perl` script. disable-interpreters.inc blocks the Perl interpreter, so it looks like you need to `include allow-perl.inc ` in a `man.local` file to support printing to PDF in the man sandbox.

Fixed in:

* #6143

**Reply by vbauerster** [reply]
Thanks a lot! Adding `include allow-perl.inc` fixed the issue.
